www-apps/otrs/otrs-5.0.15.ebuild

```diff
@@ -1,21 +1,21 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-inherit eutils confutils user systemd
+inherit confutils user systemd
 
 DESCRIPTION="OTRS is an Open source Ticket Request System"
-HOMEPAGE="http://otrs.org/"
-SRC_URI="http://ftp.otrs.org/pub/${PN}/${P}.tar.bz2"
+HOMEPAGE="https://otrs.org/"
+SRC_URI="https://ftp.otrs.org/pub/${PN}/${P}.tar.bz2"
 
 LICENSE="AGPL-3"
 KEYWORDS="~amd64 ~x86"
 IUSE="apache2 fastcgi +gd ldap mod_perl +mysql pdf postgres soap"
 SLOT="0"
 
-DEPEND="media-libs/libpng"
+DEPEND="media-libs/libpng:0="
```

Comment:
DEPEND="media-libs/libpng:0*"

leads to

Error(s) in metadata for 'www-apps/otrs-5.0.15':
  DEPEND: Invalid atom (media-libs/libpng:0*), token 1